requirements/edx/base.in

```diff
@@ -62,7 +62,7 @@ django-splash
 django-statici18n
 django-storages
 django-user-tasks
-django-waffle==0.12.0
+django-waffle==0.14.0
```

Comment:
This doesn't match the version used in the other files.  Also, are there any blockers to using the latest version `0.18.0` instead?

Reply:
Thanks for the catch, I've noticed that too, working on.